os/arch/arm/src/common/up_vfork.c

```diff
@@ -149,7 +149,7 @@ pid_t up_vfork(const struct vfork_s *context)
 	child = task_vforksetup((start_t)(context->lr & ~1));
 	if (!child) {
 		sdbg("ERROR: task_vforksetup failed\n");
-		return (pid_t)ERROR;
+		return (pid_t) ERROR;
```

Comment:
Need to revert this file all. Nothing is valid in this file.

Reply:
I will revert it